### ISSUE TITLE
keyscan test failing

### ISSUE THREAD

**Issue by christianpaquin** [open-quantum-safe/openssh-portable#47]
The keyscan test (invoked when running `make tests`) fails for the OQS algs.

@dstebila, @xvzcf, can you repro this?

**Comment by christianpaquin** [comment]
I created PR #48 to run `make tests` in CI, and it is also failing the keyscan test.

**Comment by xvzcf** [comment]
I reproduced it yesterday; The problem is that the path to `ssh-keyscan` in the `keyscan.sh` test is set by default in `test-exec.sh` to just `ssh-keyscan` (as seen [here](https://github.com/open-quantum-safe/openssh-portable/blob/OQS-master/regress/test-exec.sh#L74)), which means it is the system version that is being tested, not the one we built.

The paths to the `ssh`, `sshd`, etc. executables are also set similarly. I'll fix this next week if it isn't urgent.

**Comment by christianpaquin** [comment]
Perhaps we should move the `make tests` instruction after the `make install` in the script.

**Comment by xvzcf** [comment]
We could do that, provided we pass the install `${PREFIX}` to `make tests` as well.

**Comment by xvzcf** [comment]
I've deleted my earlier comments as I was wrong about the cause of the failures. It is not a path issue as I thought earlier, and I will provide further updates as I make progress.

**Comment by dstebila** [comment]
Check this after #53

**Comment by xvzcf** [comment]
Superseded by https://github.com/open-quantum-safe/openssh/issues/89
